packages/service/lib/start.js

```diff
@@ -2,6 +2,7 @@
 
 const { readFile } = require('fs/promises')
 const close = require('close-with-grace')
+const { Bus } = require('@platformatic/bus')
```

Comment:
This should be added to the dependencies of `@platformatic/service`

Reply:
Done